crates/server/src/graphql/modules/user/query/mod.rs

```diff
@@ -15,4 +20,18 @@ impl UserQueryRoot {
     async fn me(&self, ctx: &Context<'_>) -> Result<Me> {
         Me::exec(ctx).await
     }
+
+    async fn user(
+        &self,
+        ctx: &Context<'_>,
+        after: Option<Pxid>,
+        before: Option<Pxid>,
+        first: Option<i32>,
+        last: Option<i32>,
+        id: Option<Pxid>,
+        email: Option<String>,
+        username: Option<String>,
```

Comment:
Can we use an input here? It would be good to have a `UsersFilter` GraphQL Input with these fields.

Here are some details on GraphQL Inputs if you need them!

https://async-graphql.github.io/async-graphql/en/define_input_object.html